electrician/demo/project_task.xml

```diff
@@ -0,0 +1,90 @@
+    <field name="project_id" ref="project_project_5"/>
+    <field name="stage_id" ref="industry_fsm.planning_project_stage_0"/>
+    <field name="user_ids" eval="[(6, 0, [])]"/>
+    <field name="allocated_hours">1.0</field>
+    <field name="sale_line_id" ref="sale_order_line_21"/>
+    <field name="partner_id" ref="res_partner_14"/>
+    <field name="company_id" ref="base.main_company"/>
+  </record>
+  <record id="project_task_4" model="project.task">
+    <field name="name">S00002 - Fixed Price</field>
+    <field name="description"><![CDATA[<p> <b>Troubleshooting &amp; Repair of Power Outages</b></p><p>	•	Respond to customer complaints regarding power loss.</p><p>	•	Inspect the breaker panel, wiring, and outlets for faults.</p><p>	•	Identify the cause (e.g., tripped breaker, faulty wiring, overloaded circuit).</p><p>	•	Perform necessary repairs or replacements.</p><p>	•	Test and verify restored power, ensuring safety compliance.</p>]]></field>
+    <field name="project_id" ref="project_project_4"/>
+    <field name="stage_id" ref="industry_fsm.planning_project_stage_3"/>
+    <field name="date_end">2025-02-18 15:58:50</field>
+    <field name="user_ids" eval="[(6, 0, [])]"/>
+    <field name="sale_line_id" ref="sale_order_line_12"/>
+    <field name="partner_id" ref="res_partner_24"/>
+    <field name="company_id" ref="base.main_company"/>
+  </record>
+  <record id="project_task_3" model="project.task">
+    <field name="name">S00001 - Fixed Price</field>
+    <field name="description"><![CDATA[<p> <b>Installation of New Light Fixtures &amp; Switches</b></p><p>	•	Assess the area for fixture placement and electrical load.</p><p>	•	Turn off power and safely remove old fixtures or switches.</p><p>	•	Install new light fixtures, dimmer switches, or smart controls.</p><p>	•	Ensure secure wiring connections and mount fixtures properly.</p><p>	•	Test operation and provide instructions to the customer.</p>]]></field>
+    <field name="project_id" ref="project_project_3"/>
+    <field name="stage_id" ref="industry_fsm.planning_project_stage_1"/>
+    <field name="state">1_done</field>
+    <field name="user_ids" eval="[(6, 0, [])]"/>
+    <field name="sale_line_id" ref="sale_order_line_4"/>
+    <field name="partner_id" ref="res_partner_23"/>
+    <field name="company_id" ref="base.main_company"/>
+  </record>
+  <record id="project_task_2" model="project.task">
+    <field name="name">Meeting</field>
+    <field name="stage_id" ref="hr_timesheet.internal_project_default_stage"/>
+    <field name="user_ids" eval="[(6, 0, [ref('base.user_root')])]"/>
+    <field name="company_id" ref="base.main_company"/>
+  </record>
+  <record id="project_task_1" model="project.task">
+    <field name="name">Training</field>
+    <field name="stage_id" ref="hr_timesheet.internal_project_default_stage"/>
+    <field name="user_ids" eval="[(6, 0, [ref('base.user_root')])]"/>
+    <field name="company_id" ref="base.main_company"/>
+  </record>
```

Comment:
Where are these two tasks coming from? I think those are automatically created (and not useful). Could you check?
```suggestion
```